CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 13.2.3 (07/19/23)
+
+* Automatic Upgrades
+  * Upgrader monitoring and alerts [#29206](https://github.com/gravitational/teleport/pull/29206)
+* CLI
+  * AWS DBs Heartbeat: return IAM status (#28952) [#29196](https://github.com/gravitational/teleport/pull/29196)
+  * Provide warning on tsh sso login with Teleport user specified [#29221](https://github.com/gravitational/teleport/pull/29221)
+* Hosted Plugins
+  * Restores default API endpoint for PagerDuty plugin [#29295](https://github.com/gravitational/teleport/pull/29295)
+* Kubernetes Access
+  * Fix proxy protocol support for Kube access flow [#29274](https://github.com/gravitational/teleport/pull/29274)
```

Comment:
@tcsc PR titles as-is are most of the time not suitable for the changelog, and formatted all over the place. I usually try to skim through the PR description and rephrase the PR title so readers understand what is being fixed or implemented. For example, `AWS DBs Heartbeat: return IAM status (#28952)` is not a great title to get as-is.

Also things like internal product metrics are not relevant to users so I exclude things like that as well.